_layouts/default.html

```diff
@@ -6,7 +6,6 @@
 <head>
 <!-- Global site tag (gtag.js) - Google Analytics -->
 <script async src="https://www.googletagmanager.com/gtag/js?id=UA-82811140-33"></script>
-<script async src="https://cdn.jsdelivr.net/npm/details-polyfill@1/index.min.js"></script>
```

Comment:
Can we add this back in? It was intended to solve a challenge with the <details> element not rendering correctly on IE 10 and older.